nautobot/core/views/mixins.py

```diff
@@ -992,10 +992,9 @@ def send_bulk_delete_objects_to_job(self, request, pk_list, model, delete_all):
         )
         return redirect("extras:jobresult", pk=job_result.pk)
 
-    def send_bulk_edit_objects_to_job(self, request, form, model):
+    def send_bulk_edit_objects_to_job(self, request, form_data, model):
```

Comment:
This is *technically* a breaking change to a public API, though from a quick github search I don't see where any apps have adopted/overridden it yet since it's new in 2.4.0.

Reply:
Thank you for the research